src/assets/sass/protocol/base/_elements.scss

```diff
@@ -10,3 +10,4 @@
 @import 'elements/tables';
 @import 'elements/forms';
 @import 'elements/sections';
+@import 'elements/animations';
```

Comment:
I know animations don't really tie directly to an element, but this felt like the most logical place to put this, without making a new directory under `/base/`. Happy to change this though